Header header

GoalID goal_id

RequestType request_type

MoveBaseGoal goal
